src/redist/targets/Crossgen.targets

```diff
@@ -48,6 +48,11 @@
       <!-- Add back the .NET Core assemblies in the Sdks folder -->
       <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk\tools\netcoreapp*\**\*" />
       <RemainingFiles Include="$(SdkOutputDirectory)Sdks\NuGet.Build.Tasks.Pack\CoreCLR\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Razor\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Web\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.WindowsDesktop\tools\netcoreapp*\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\ILLink.Tasks\tools\netcoreapp*\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Publish\tools\netcoreapp*\**\*" />
```

Comment:
We may add more Sdks over time.  It would be nice to have a process to keep this list up-to-date.

Reply:
Yes, I'm not sure what the right thing to do is here - one option is to convert this to a more broad "crossgen everything" approach and then work through things that we don't want to crossgen (or don't work).  I attempted this early on but was having trouble tracking down what exactly was breaking, and so I went with this approach.  I'm also wondering if there is work in the optional components arena to make sure that things are crossgen'd as well.

Reply:
Good point for optional components. Components need to cross-gen in their repo since sdk will not rebuild when there is a new component available